### ISSUE TITLE
Angular version numbers in peerDependencies for mf-runtime

### ISSUE THREAD

**Issue by tlinqu** [angular-architects/module-federation-plugin#72]
In https://github.com/angular-architects/module-federation-plugin/blob/main/libs/mf-runtime/package.json,

```json
"peerDependencies": {
    "@angular/common": ">=12.4.0",
    "@angular/core": ">=12.4.0"
  }
```

The version numbers are mis-typed.

**Comment by manfredsteyer** [comment]
Thanks for pointing this out. Yes, we need to fix this. If you use the share helper in the meantime or set requiredVersion by hand, everything should work though.

**Comment by gcollin** [comment]
Sorry for my stupidity but how can I use requiredVersion to add it to an existing project?

**Comment by manfredsteyer** [comment]
Thanks. Just fixed it with version 12.4.1
